billing/scripts/helpers.js

```diff
@@ -0,0 +1,66 @@
+import Big from 'big.js'
+import { StoreOperationFailure } from '../tables/lib.js'
+import { EndOfQueue } from '../test/helpers/queue.js'
+
+/**
+ * @template T
+ * @returns {import('../lib/api.js').QueueAdder<T> & import('../test/lib/api.js').QueueRemover<T>}
+ */
+export const createMemoryQueue = () => {
+  /** @type {T[]} */
+  const items = []
+  return {
+    add: async (message) => {
+      items.push(message)
+      return { ok: {} }
+    },
+    remove: async () => {
+      const item = items.shift()
+      return item ? { ok: item } : { error: new EndOfQueue() }
+    }
+  }
+}
+
+/**
+ * @template T
+ * @returns {import('../lib/api.js').StorePutter<T> & import('../lib/api.js').StoreLister<any, T> & import('../lib/api.js').StoreGetter<any, T>}
+ */
+export const createMemoryStore = () => {
```

Comment:
I assume based on your ran that it does not load a lot of memory in the host running. wondering if we should write in disk to avoid getting out of memory, but probably nothing close to happen that we can start with in memory?